crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -325,6 +312,22 @@ impl From<&SlidingSyncRoom> for FrozenSlidingSyncRoom {
     }
 }
 
+impl From<(FrozenSlidingSyncRoom, Client)> for SlidingSyncRoom {
```

Comment:
I'm not a fan of this change. `From` impls on tuples are a bit magic. Also this makes the conversion part of the public interface, it was private before.

Reply:
OK, let's rollback to what we had previously.